### PR TITLE
Create fromBytes and test implementation

### LINKED ISSUE #8506: create and test `fromHexString` for `SszBitVector`
As a developer using the `SszBitVector` class, 
it would be convenient to have `fromHexString` and `fromBytes` defined,
So that I can create an `SszBitVector` from a hex or bytes payload.

Basically, `fromBytes` would do the heavy lifting, `fromHexString` would convert to bytes then call `fromBytes`.

I'd suggest following how `ofBits` is done basically.

The implementation will likely be in `SszBitVectorImpl` where `ofBits` is defined currently. and would be on the `SszBitVectorSchema`

Testing will be in SszBitVectorTest.

### PR DESCRIPTION
**Pull request by gconnect**
## PR Description
- [x] Create and test fromBytes
- [x] Create and test fromHexString

## Fixed Issue(s)
fixes #8506

### CODE REVIEW

**Comment by rolfyone** [comment]
thanks for the pr, can you run `./gradlew spotlessJavaApply` ? it'll clean up the whitespace and make the code comparable..

**Comment by gconnect** [comment]
> thanks for the pr, can you run `./gradlew spotlessJavaApply` ? it'll clean up the whitespace and make the code comparable..

I just ran it now and pushed update.

File: infrastructure/ssz/src/test/java/tech/pegasys/teku/infrastructure/ssz/collections/SszBitvectorTest.java [hunk 1]
```diff
@@ -239,4 +240,13 @@ void testGetLastSetBitIndex(final SszBitvector bitvector) {
   void get_shouldThrowIndexOutOfBounds(final SszBitvector vector) {
     assertThatThrownBy(() -> vector.get(-1)).isInstanceOf(IndexOutOfBoundsException.class);
   }
+
+  @ParameterizedTest
+  @MethodSource("bitvectorArgs")
+  void testFromBytes(final SszBitvector bitvector) {
+    Bytes bytes = bitvector.sszSerialize();
+    long size = bitvector.getSchema().getMaxLength();
+    SszBitvector result = SszBitvectorImpl.fromBytes(bitvector.getSchema(), bytes, (int) size);
```
Comment (rolfyone) [inline infrastructure/ssz/src/test/java/tech/pegasys/teku/infrastructure/ssz/collections/SszBitvectorTest.java]: smallest of nits, lets make these variables final and then we're good here i think...

Comment (rolfyone) [reply infrastructure/ssz/src/test/java/tech/pegasys/teku/infrastructure/ssz/collections/SszBitvectorTest.java]: can always fix the whole file, they should all really be final :)

**Comment by gconnect** [comment]
@rolfyone I have made the changes!

**Review by rolfyone — APPROVED** [review]
LGTM - thanks for the PR, I'll merge it shortly!

**Comment by rolfyone** [comment]
@KoxyG can you sign the CLA?

**Comment by gconnect** [comment]
I have also pushed update for the part she was supposed to work on.

**Comment by rolfyone** [comment]
closing in favour of #8573

**Pull request closed without merging**